mantidproject/mantid
# MR filtering for cross-section extraction

With the MR (REF_M) DAS upgrade, the cross-sections will be extracted from the data using log entries that define the initial and final polarization states.
Before the upgrade, the four cross-sections were saved in separate entries in the nexus file.

We need a filtering algorithm that will load a file and inspect the logs to extract the cross-sections, which will be returned in a workspace group. Since we need the reduction to be backward compatible, the algorithm needs to be able to process the legacy format too.

**To test:**
- Make sure the tests pass.
- Inspect code.

There is no "issue" for this PR, all the info is here.

**Release Notes** 
A note has been added to the reflectivity section of the 3.12 release notes.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
